### PR TITLE
Hide avatar from screenreaders

### PR DESCRIPTION
**Pull request by awikkerink**

### CODE REVIEW

**Comment by omsmith** [comment]
_Are_ profile images decoractive? Do we treat them as deciratuve in the rest of the LMS? I feel like _Profile image for ${name}_ as an alt is what I would do, but whatever is consistent

**Comment by awikkerink** [comment]
The aria-label on https://github.com/Brightspace/user-tile/pull/52/files#diff-c1a2647e4679e9ed3c7eb3a0ace0fd20R97 should be enough? I suppose having "profile image for [blank]" could also work. And `alt` might be more correct...

**Comment by omsmith** [comment]
Yeah meh, hiding it seems fine. Not like we can _actually_ describe what's in the image. It may or may not be their face, it might be a funster, it might be a dog, or a fish, or a manatee.

File: test/d2l-user-tile/d2l-user-tile-a11y.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+			</template>
+		</test-fixture>
+
+		<test-fixture id="with-user-data">
+			<template>
+				<d2l-user-tile
+					icon="testimage.jpg"
+					name="Lisa Simpson"
+					grade="4th Grade"
+					background-color:"#0000FF">
```
Comment (ryantmer) [inline test/d2l-user-tile/d2l-user-tile-a11y.html]: This should be `=` rather than `:` yeah?

Comment (awikkerink) [reply test/d2l-user-tile/d2l-user-tile-a11y.html]: You are correct. I just did a copy+paste

Comment (awikkerink) [reply test/d2l-user-tile/d2l-user-tile-a11y.html]: Maybe this was supposed to be part of the style attribute? Doesn't seem to matter

Comment (ryantmer) [reply test/d2l-user-tile/d2l-user-tile-a11y.html]: Yeah I definitely did a double-take, thought it was `style` :)

**Pull request merged**